python/paddle/fluid/layers/nn.py

```diff
@@ -10560,3 +10561,52 @@ def tree_conv(nodes_vector,
     else:
         pre_activation = out
     return helper.append_activation(pre_activation)
+
+
+def npair_loss(anchor, positive, labels, l2_reg=0.002):
+    '''
+  **Npair Loss Layer**
+  
+  see http://www.nec-labs.com/uploads/images/Department-Images/MediaAnalytics/papers/nips16_npairmetriclearning.pdf
```

Comment:
To make it more elegent, use indirect link in the manner of:
```suggestion
  Read `Improved Deep Metric Learning with
Multi-class N-pair Loss Objective <http://www.nec-labs.com/uploads/images/Department-Images/MediaAnalytics/papers/nips16_npairmetriclearning.pdf>`_ .
```
Could you please preview the effect again to make sure the link is correct and elegent.